browser.js

```diff
@@ -98,27 +98,33 @@ function changeTab(next) {
 	$(`a[href$="${pages[ret]}"]`).click();
 }
 
+// sets interval to wait for selector to be ready before firing callback
+function waitFor(selectorString) {
+	return new Promise(resolve => {
+		const awaitElement = setInterval(() => {
```

Comment:
To do this outside of the `setInterval` I have to turn this into a function. A regular assignment won't change on each interval, causing an infinite loop of the initial state on load.